.circleci/config.yml

```diff
@@ -24,10 +24,10 @@ jobs:
           command: npm run build
       - run:
           name: Tests
-          command: npm run test:ci
+          command: npm run test:ci -- -- --maxWorkers=1
```

Comment:
Could `--maxWorkers` be provided in the `test:ci` script rather than here?

Reply:
Sure [c24a582](https://github.com/auth0/node-auth0/pull/898/commits/c24a582dc2ad832d8ef4c097191ed3e8b6fb656f) 👍